src/handlers/GraphQLHandler.test.ts

```diff
@@ -3,7 +3,7 @@
  */
 import { encodeBuffer } from '@mswjs/interceptors'
 import { OperationTypeNode, parse } from 'graphql'
-import { Headers } from 'headers-polyfill/lib'
+import { Headers } from 'headers-polyfill/lib/index.js'
```

Comment:
When I get some time, exposing an esm build of headers-polyfill should be pretty quick, and might be better than importing from lib directly here

Reply:
Yes this would be the cleanest solution. Until then this would be an easy workaround

Reply:
just opened https://github.com/mswjs/headers-polyfill/pull/38

Reply:
`headers-polyfill@3.1` ships with ESM support now!

I've updated `@mswjs/interceptors` to use it, now updating MSW as well.